pro.affine.affine.yaml

```diff
@@ -0,0 +1,47 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/refs/heads/main/data/flatpak-manifest.schema.json
+app-id: pro.affine.affine
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: start-affine.sh
+separate-locales: false
+rename-desktop-file: affine.desktop
+rename-appdata-file: affine.metainfo.xml
+rename-icon: affine
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
```

Comment:
why is network and pulseaudio needed?

Reply:
This editor has synchronization capabilities, so I'm pretty sure network is needed. As for pulseaudio - not really sure, just copied config from their electron-forge settings. Is it required for audio playback? Because there are video embeds and such. I'll try disabling it and test if it works after work today

Reply:
Tried disabling pulseaudio socket - crashes on video playback. Works fine while it's enabled. So I guess it is needed